packages/utils/telemetry-utils/package.json

```diff
@@ -153,6 +153,9 @@
 			"RemovedFunctionDeclaration_isValidLegacyError": {
 				"forwardCompat": false,
 				"backCompat": false
+			},
+			"ClassDeclaration_MockLogger": {
+				"forwardCompat": false
```

Comment:
This went to the other PR, right?

Reply:
Urgh, apparently not. I guess I only ported over the code changes :/